agents/unix/conf/base/conf_cpu.c

```diff
@@ -895,6 +899,73 @@ cpu_thread_grab(const char *name)
     return rc;
 }
 
+static te_errno
+memory_get(unsigned int gid, const char *oid, char *value)
+{
+  long int n_pages, page_size;
+  uint64_t mem;
+
+  UNUSED(gid);
+  UNUSED(oid);
+
+  errno = 0;
+
+  n_pages = sysconf(_SC_PHYS_PAGES);
```

Comment:
`_SC_PHYS_PAGES` is not in POSIX, so it's needed to check for its presence with `#ifdef _SC_PHYS_PAGES` (and, I believe, just return `0` if it is not present)